lighthouse-core/audits/byte-efficiency/uses-webp-images.js

```diff
@@ -21,9 +21,9 @@ class UsesWebPImages extends ByteEfficiencyAudit {
   static get meta() {
     return {
       name: 'uses-webp-images',
-      description: 'Serve images as WebP',
+      description: 'Serve images in next-gen formats',
       informative: true,
-      helpText: 'WebP provides better lossy and lossless compression than PNG or JPEG, ' +
+      helpText: 'JP2, JPEG XR, and WebP often provide better compression than PNG or JPEG, ' +
```

Comment:
let's use the elongated names

> Image formats like JPEG 2000, JPEG XR, and WebP often provide better compression than PNG or JPEG

Reply:
ya, done

Reply:
Passing comment: as part of the research we did for images.guide, it turned out that a lot of users could get some pretty hefty wins from adopted MozJPEG. This isn't technically a next-gen format, but a JPEG optimizer. Unsure if you'd want to cater for those types of recommendations.